### ISSUE TITLE
Ability to specify CouchDB image and version

### ISSUE THREAD

**Issue by mrabe142** [kfsoftware/hlf-operator#86]
I host containers in my own registry since my systems do not have direct access to the internet. I can specify the image and version for the fabric ca, fabric peer, fabric orderer, and chaincode. There does not appear to be a way to point the peer deploy to pull the couchdb image from my registry like all the other images.

**Comment by dviejokfs** [comment]
Hi @mrabe142 

You can specify in the v1.6.0 version the image like this:

```yaml
apiVersion: hlf.kungfusoftware.es/v1alpha1
kind: FabricPeer
metadata:
  creationTimestamp: null
  name: org1-peer0
  namespace: default
spec:
# ...more props
  couchdb:
    password: couchdb
    user: couchdb
    image: yourownreg
    tag: yourtag
    externalCouchDB:
      enabled: true # "true" to use external couchdb, "false" to have it as a sidecar
      host: <EXTERNAL_COUCHDB_HOST>
      port: <EXTERNAL_COUCHDB_PORT>
# ...more props
```

The only issue would be to pass the image pull secrets to pull the image

**Comment by dviejokfs** [comment]
Implemented in 1.7.0

https://labs.hyperledger.org/hlf-operator/docs/operator-guide/state-db#configuring-couchdb
